core/src/main/java/org/ehcache/spi/cache/tiering/LowerCachingTier.java

```diff
@@ -38,7 +38,7 @@
    * @return the value holder, or null.
    * @throws CacheAccessException
    */
-  Store.ValueHolder<V> getOrComputeIfAbsent(K key, Function<K, Store.ValueHolder<V>> source) throws CacheAccessException;
+  Store.ValueHolder<V> installMapping(K key, Function<K, Store.ValueHolder<V>> source) throws CacheAccessException;
```

Comment:
Why are we renaming this method in this PR?  And if it's necessary could we do the rename in it's own commit?